### PR TITLE
Ms dict updates

### PR DESCRIPTION
**Pull request by Sjhunt93**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -299,6 +299,9 @@ Speech methods are:
     - `audience` (dict) - Specify the values of parameters in your script. For instance, if in the script resource you have `scriptText="Hello {{name}} {{lastname}}, welcome to {{location}}"`, the audience should be: `{"name": "Elon", "lastname": "Musk", "location": "Istanbul"}`. If not provided, the fallback track will be created.
     - `sync` (boolean) - Allow sync or async speech creation. Default is `True`. If `sync=False`, speech create call will return a success message when the speech creation is triggered. To retrieve the files, check `Speech.retrieve()` method.
     - `sections` (dict) - Specify parameters for specific sections in the script. The key is a section name, and the value is another dictionary with the section configuration ( valid parameters are: voice, speed, effect, silence_padding). If a section is not found here, the section will automatically inherit the voice, speed, effect and silence_padding values you defined above (or the default ones if you don't provide them). See an example below with 2 sections and different configuration parameters being used.
+    - `useDictionary` (bool) - Applies lexi to the script text.
```
Comment (LindaKivine) [inline README.md:302]: We are calling it Pronunciation Dictionary now. Lets try to get rid of lexi as much as possible

**Review by LindaKivine — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (LindaKivine) [inline README.md:302]:
```suggestion
    - `useDictionary` (bool) - Applies pronunciation dictionary to the script text.
```

File: README.md [hunk 2]
```diff
@@ -653,11 +656,16 @@ Lexi is an engine for enhancing the pronunciation of troublesome words. For exam
 
 To use this feature words in the script should be marked up with the `<!'type'>` flag, whereby type is the type of dictionary to use. The dictionary flag that precedes the word should contain the type, and the one following should be empty `<!>`. In the example shown below, the second occurrence of the word **reading** will be pronounced as the city name.
 
+To use lexi end-to-end ensure that you supply a voice whose language matches that of the language of the dictionary you wish to use, and then supply `useDictionary=true` to the speech create call (see the example below). See the [Voice](#voice) resource to get a list of voices and their corresponding language code, or browse our available voices [here](#https://library.api.audio/voices)
```
Comment (LindaKivine) [inline README.md:659]:
```suggestion
To use pronunciation dictionary end-to-end ensure that you supply a voice whose language matches that of the language of the dictionary you wish to use, and then supply `useDictionary=true` to the speech create call (see the example below). See the [Voice](#voice) resource to get a list of voices and their corresponding language code, or browse our available voices [here](#https://library.api.audio/voices)
```

File: README.md [hunk 3]
```diff
@@ -653,11 +656,16 @@ Lexi is an engine for enhancing the pronunciation of troublesome words. For exam
...
-    scriptText = "Hello I am reading a book in the city of <!location>reading<!> today"
-    ```
+  ```python
+  scriptText = """Hello I am reading a book in the city of <!location>reading<!> today"""
+  script = apiaudio.Script.create(scriptText=scriptText)
+  speech = apiaudio.Speech.create(scriptId=script["scriptId"], voice="Ryan", useDictionary=True)
+  print(speech)
+  ```
 
 Lexi methods are:
```
Comment (LindaKivine) [inline README.md:670]:
```suggestion
Prononciation dictionary methods are:
```

**Review by LindaKivine — CHANGES_REQUESTED** [review]
updated lexi to pronunciation dictionary

**Review by springcoil — APPROVED** [review]
LGTM

**Review by LindaKivine — APPROVED** [review]

**Pull request merged**